src/modding/ModManager.cs

```diff
@@ -857,6 +1308,251 @@ private void OpenInfoUrlPressed()
         }
     }
 
+    private void OnDependencyPressed()
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+        if (selectedMod is null)
```

Comment:
Use normal null checks (I'll stop commenting about this now) instead of `is null` everywhere where this is used.